datafusion/physical-expr/src/window/nth_value.rs

```diff
@@ -77,17 +79,17 @@ impl NthValue {
         n: u32,
```

Comment:
why not change  this API  to accept i64? It seems strange that the public interface doesn't support negative NTH valuess

Reply:
Postgre doesn't have this feature. I thought it might be non-standard or unexpected. However we can this support in subsequent PRs.